src/User/User.php

```diff
@@ -564,7 +570,12 @@ public function setPreference($key, $value)
      */
     public function updateLastSeen()
     {
-        $this->last_seen_at = Carbon::now();
+        $now = Carbon::now();
+
+        // Only update if the current timestamp is from a while ago
```

Comment:
Let's drop this comment and the correspondone one in AccessToken.php. Otherwise good to merge!

Reply:
```suggestion
```